src/app/dashboard/page.tsx

```diff
@@ -83,6 +83,21 @@ export default function Home() {
                       <Td>
                         <Checkbox
                           isChecked={user.admin == "0" ? false : true}
+                          onChange={(e) => {
```

Comment:
minor: we can handle `onChange` event outside of component.

Reply:
if I typed the code outside of Checkbox, I must define the e's type...
so i define the function in the checkbox's onChange handler.